design-proposals/golden-image-delivery-and-update-pipeline.md

```diff
@@ -72,15 +72,19 @@ The common templates shipped with SSP need to have their DataVolumeTemplate sect
 
 ## HCO: Integration with SSP Operator
 
-During installation, the HCO bundle's CSV will pass into the HCO operator (as ENV VARs) locations where disk images are sourced from for the automated disk image delivery/update feature. The end user can opt-in to automatic delivery/updates of these image sources through a tuning on the HCO's CR. From there the HCO is responsible for configuring DataImportCrons on the SSP operator's CR.
+During installation, the HCO bundle's CSV will pass into the HCO operator (as a local yaml file within the HCO image) locations where disk images are sourced from for the automated disk image delivery/update feature. 
 
-This document doesn't include within it's scope how the HCO should expose this feature to the end user. The general expectation is that we will start with a simple tuning that enables toggling automated delivery/updates on and then consider advanced tunings in future releases. The exact structure of where this tuning belongs on the HCO API is left up to the HCO community.
+The end user can opt-in to automatic delivery/updates of these image sources through a tuning on the HyperConverged CR, by setting the `commonDataImportCronEnabled` feature gate to `true`. From there the HCO is responsible for configuring DataImportCrons on the SSP operator's CR.
```

Comment:
The DataImportCron name is related to the underlying CRD that facilitates the import of updated data to a PVC.

Are you all sure you want to expose that naming as the HCO tuning? The feature being enabled here is more directly related to importing common base images. Would something like "enableCommonBootImageImport" make more sense?

Reply:
Thank @davidvossel, that makes sense. I'll rename the feature gate.